src/SFML/Window/GlContext.cpp

```diff
@@ -73,9 +68,38 @@
 
 #endif
 
+#if !defined(GL_MULTISAMPLE)
```

Comment:
Is there a specific reason why those macros are defined here and not in `GLLoader.hpp`?